### DISCUSSION TITLE
Email request to "borrow" my github account for 10 minutes

### DISCUSSION THREAD

**Discussion by leres** [community/community#110587] [New to GitHub]
I received a request via email (sent to an address only known to github!) to "borrow" my account in exchange for 1200 yuan. This seemed worth reporting (in a timely manner) but after poking at it for more than an hour I can't figure out how to do it. Perhaps interested parties could reach out to me?

**Comment by Akash1134** [comment]
Hi @leres , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Comment by leon-barrett** [comment]
I got a similar email, and I couldn't find how to use any of those reporting tools to report an email about GitHub that I got from an unrelated email address.

This is a different sort of abuse than those documents address. They show how to deal with abuse that I'm receiving on GitHub, but not how to warn GitHub that people are trying to abuse the service itself.

**Reply by leres** [reply]
Exactly! I had already found and discarded the links in @Akash1134's post.

**Comment by ThomasWaldmann** [comment]
Also got that email. Did they spam / try to scam everybody?

"""
Dear friend，

I am a Chinese developer live in Chengdu, your github account has received the Starknet airdrop. If you are familiar with the operation, you can collect it yourself and exchange it for cash. If you are not familiar with the relevant operation, I can directly give you 1,200 yuan in cash or 180 US dollars via paypal/alipay and use your account for 10 minutes.
I run a mining pool and am familiar with the gameplay of the crypto circle. ...
"""

**Reply by angry-mustard-it** [reply]
Apparently, that came from a _ElsaGate_-like phenomenon about Chinese, Indian and Vietnam sellers/bitcoin miners/scams.

ElsaGate created account hackings on YouTube and seizure-inducing content that is not appropriate on YouTube (gore, graphic imagery, car crashes, all on YouTube Shorts)

**Comment by leres** [comment]
A friend found info that likely explains the attack:

[https://therecord.media/github-investigating-crypto-mining-campaign-abusing-its-server-infrastructure](https://therecord.media/github-investigating-crypto-mining-campaign-abusing-its-server-infrastructure)

> The attack involves forking a legitimate repository, adding malicious GitHub Actions to the original code, and then filing a Pull Request with the original repository in order to merge the code back into the original.
> 
> But the attack doesn't rely on the original project owner approving the malicious Pull Request. Just filing the Pull Request is enough for the attack, Perdok said.
> 
> The Dutch security engineer told us attackers specifically target GitHub project owners that have [automated workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows) that test incoming pull requests via automated jobs.

(Too bad that so far github doesn't have a method for us to report attempts...)

**Reply by ThomasWaldmann** [reply]
Guess that's why you have to approve PRs from new contributors before it'll run the actions.
